app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_4">4dp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="sp_32">32sp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="sp_16">16sp</dimen>
+    <dimen name="dp_480">480dp</dimen>
+    <dimen name="dp_96">96dp</dimen>
+    <dimen name="dp_48">48dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_48` appears to be unused
 Unused resources make applications larger and slow down builds.